src/state/nftMarket/helpers.ts

```diff
@@ -27,3 +27,27 @@ export const getCollections = async (): Promise<any[]> => {
     return []
   }
 }
+
+export const getNftsFromCollection = async (collectionAddress: string): Promise<any[]> => {
+  try {
+    const res = await request(
+      GRAPH_API_NFTMARKET,
+      gql`
+        {
+          nfts(where: { collection: "${collectionAddress}" }) {
+            id
+            currentSeller
+            isTradable
+            tokenId
+            metadataUrl
+          }
```

Comment:
Should we put GraphQL queries in a separate folder ?

Reply:
it's already in a separate file